tools/wpt/browser.py

```diff
@@ -1559,7 +1559,23 @@ def install_webdriver(self, dest=None, channel=None, browser_binary=None):
         raise NotImplementedError
 
     def version(self, binary=None, webdriver_binary=None):
-        return None
+        if webdriver_binary is None:
+            self.logger.warning(
+                "Cannot find ChromeiOS version without webdriver_binary")
```

Comment:
Instead of webdriver_binary, wouldn't ChromeDriver or CWTChromeDriver be more accurate and intuitive here?

Reply:
Thanks, updated